R/navbar.R

```diff
@@ -2,13 +2,18 @@ data_navbar <- function(pkg = ".", depth = 0L) {
   pkg <- as_pkgdown(pkg)
 
   navbar <- config_pluck(pkg, "navbar")
-
-  style <- navbar_style(
-    navbar = navbar,
-    theme = get_bslib_theme(pkg),
-    bs_version = pkg$bs_version
-  )
-
+  
+  uses_lightswitch <- !is.null(config_pluck_string(pkg, "template.theme-dark"))
```

Comment:
I'm pretty sure that using bootswatch and/or navbar colour customisation is incompatible with the lightswitch, so I need to add some checks for that somewhere.